### ISSUE TITLE
The aria-keyshortcuts is wrongly attached for the ActionList.LinkItem components

### ISSUE THREAD

**Issue by jchuerva** [primer/react#2411]
From https://github.com/github/repos/issues/3034

It looks like for `ActionList.LinkItem` component, the `aria-keyshortcuts` is attached the the `<a>` instead of `<li>` which is incorrect.

![image](https://user-images.githubusercontent.com/5942864/194352716-0fe7c774-1651-4470-b5f8-1326b2f1b088.png)

**Comment by lesliecdubs** [comment]
Thanks for reporting this bug, @jchuerva! This looks like it'll require a slightly more complex fix so we're moving it to our backlog for now but hope to revisit soon.

**Comment by siddharthkp** [comment]
Leaving here for who would work on this: After moving aria-keyshortcuts, need to make sure the link can still be opened with keyboard

**Comment by lesliecdubs** [comment]
Good questions @green6erry! If you'd like an accessibility expert to weigh in on them, I'd suggest asking about this over in #accessibility on Slack or signing up for the next [Accessibility Eng+ Office Hours](https://github.com/github/accessibility/issues?q=is%3Aissue+is%3Aopen+label%3Ameeting+office+hours) for more hands-on support and advice. 

This [helpful accessibility guide](https://github.com/github/accessibility#im-looking-for-help) may come in handy for your work!

**Comment by tallys** [comment]
hey @siddharthkp, checking in on this
